docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -0,0 +1,79 @@
+---
+approvers:
+- derekwaynecarr title: Manage HugePages
+---
+
+{% capture overview %}
+
+Kubernetes supports the allocation and consumption of pre-allocated huge pages
+by applications in a pod as an **alpha** feature.  This page describes how users
+can consume huge pages and the current limitations.
```

Comment:
Our style is to capitalize Pod, Container, and other API objects.

"... by applications in a Pod ..."